### PR TITLE
Fixes #31008 - add status_link method to compliance-status

### PR DESCRIPTION
**Pull request by yifatmakias**
Compliance status on hosts/show page should be clickable and lead to the last report for the host.
In order to add this in foreman there is a need to create a status_link method in the compliance status.

### CODE REVIEW

**Comment by yifatmakias** [comment]
related to this PR:
https://github.com/theforeman/foreman/pull/8044

File: app/models/foreman_openscap/compliance_status.rb [hunk 1]
```diff
@@ -8,6 +8,10 @@ def self.status_name
       N_('Compliance')
     end
 
+    def status_link
+      host.arf_reports_path(:search => "host = #{host.id}")
```
Comment (xprazak2) [inline app/models/foreman_openscap/compliance_status.rb:12]: Should the search be `host = #{host.name}`?

**Comment by xprazak2** [comment]
One minor nit, otherwise looks good.

**Comment by xprazak2** [comment]
Works well together with the changes in core.

File: app/models/foreman_openscap/compliance_status.rb (hunk 1 above)
Comment (ares) [reply app/models/foreman_openscap/compliance_status.rb]: ir should it even link to the last report directly? note that status represents results of the last scan

Comment (xprazak2) [reply app/models/foreman_openscap/compliance_status.rb]: If host has multiple policies and several of them are failing, showing just one report might be misleading. I think we are better off showing a list of reports rather that a single one.

**Comment by xprazak2** [comment]
It great that foreman_openscap has another contributor, thanks!

**Comment by ares** [comment]
@yifatmakias great addition, would you mind demoing it? Please sign up at https://community.theforeman.org/t/foreman-community-demo-87/21229, people should see how they can improve their plugins in the same way :-)

**Comment by yifatmakias** [comment]
@ares I think it is a good idea to demo it, I will sign up for the next demo. Thanks :)

**Pull request merged**